src/app/app.module.ts

```diff
@@ -24,6 +25,23 @@ import { TermsAndConditionsModalComponent } from './components/challenge/challen
 // import module
 import { SharedModule } from './shared/shared.module';
 
+// configuration of cookie consent
+const cookieConfig: NgcCookieConsentConfig = {
+  cookie: {
+    domain: 'localhost' // or 'your.domain.com' // it is mandatory to set a domain, for cookies to work properly (see https://goo.gl/S2Hy2A)
```

Comment:
What will be the domain when it is in production or staging environment? At that time how will we configure it? Can it pick the domain without setting it - can you explore about it ?

Reply:
@Shekharrajak we just need to change the domain in here for the production mode and in it's doc, i couldn't find any feature of automatic domain detection.So will have to change it manually at the time of production.

Reply:
Try to make it environment variable then.

Reply:
Thank you for suggestion @Shekharrajak . I have made the suggested changes. Can you please now check if this looks good to you?